cmd/airflow.go

```diff
@@ -753,3 +763,13 @@ func prepareDefaultAirflowImageTag(airflowVersion string, httpClient *airflowver
 	}
 	return defaultImageTag
 }
+
+func checkPort(port string) error {
```

Comment:
Does this have to be a string or can it be an int? Also what if the string was `""` or if int was 0?

Reply:
net.Listen() takes a string so this would have to be a string